.devcontainer/codespaces-start.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
As this stands I don't think the platform team will want to merge this PR, since it would start the MHV mocks for all Codespaces users. Perhaps it could be changed to a script to start the MHV mocks for codespaces with usage documented in an MHV level readme file?

Reply:
This script will not execute anything unless whoever attempt to start up Codespaces environment will have a secret `VETS_WEBSITE_MHV_MOCK_SERVICE` configured in their Github profile (per line 3 condition below 👇🏻 ). If you think this may be alright, I should do a better job documenting this aspect. Although I was planning to create a separate documentation for our teams, I was not sure if it would be an appropriate place in `vets-website` or `va.gov-team`

Reply:
Ah thanks for the response, I completely missed that condition! I think I still lean somewhat against this approach, for a couple of reasons:
- it pulls team-specific workflows into the global codespaces configuration
- it feels more opaque to me to have to create github configuration than to run a script, but that could be a matter of taste

@mdewey what are your thoughts?

As far as documentation location, for developer-specific documentation I tend to lean towards a README in vets-website in the app root folder, but wherever people will look for it is best 😄

Reply:
> it feels more opaque to me to have to create github configuration than to run a script, but that could be a matter of taste

It was primarily inspired by [this](https://github.com/department-of-veterans-affairs/vets-website/blob/d94e669a0859bf7c7e50a42b8c1d19a0b88aa60b/.devcontainer/codespaces-create.sh#L23) and in hope to make a process more streamlined and less sophisticated for non-technical folks. But we are totally not opposed to keep it isolated on a dedicated branch without merging it. It would require regular maintenance by engineers to keep that dedicated branch updated with `main` all the time 🤔

Reply:
yeah, imo the difference between this and the build content flag is that it's not specific to a given app. The joys of living in a monorepo :-( 

Anyway I think this approach makes sense for a single team, but I'm not sure what it looks like for many app teams. Ultimately it would be good to see what @department-of-veterans-affairs/va-platform-cop-frontend folks think - options could be:

- manual script execution only
- ok to have app-specific config in the codespaces config
- build a light-weight hook system where the main post-start script scans app dirs for individual post-start scripts